src/test/java/swaglabs/tests/ui/GOOGLECODE_VIOLATE/VALIDATE_Google_Standard_Violation_12s.java

```
@@ -0,0 +1,67 @@
+package swaglabs.tests.ui.GOOGLECODE_VIOLATE;
+
+import java.io.IOException;
+
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import com.google.common.collect.ImmutableList;
+
+import groovyjarjarantlr4.v4.runtime.misc.NotNull;
+import swaglabs.common.CommonMethods;
+import swaglabs.components.DriverFactory;
+
+public class VALIDATE_Google_Standard_Violation_12s extends DriverFactory {
+
+	/**
+	 // This test is to violate google coding standdard
+	 */
+	
+	private int         x;
+	int a, b,   c,  d;
+	String[][] u = {  {"foo"}   };
+	static public final String va = null;
+	Long l = 43543543l;
+	static final ImmutableList<String> immutaBLE = ImmutableList.of("Ed", "Ann");
+	String Non__cOnstant;
+	
+	/**
+	 * This test * @param @author 2280571
+	 */
+
+	@Test
+	public void valiTheSwagLabsByLoginIntoItAndThenValidateUserIsSuccessfullyLandedIntoItAfterThatLogoutFromApplication(){
+		switch (1) {
+		case 1:
+		case 2:
+		case 3:
+			System.out.println("No default");
+			
+		
+		}
+		if(1>0)
+			Assert.assertEquals(false, false);
+		DriverFactory d = new DriverFactory();
+		try {
+			    new CommonMethods().addToCartAndVerifyItemAdded(null, null);
+			  } catch (Exception e) {}
```

As per Google coding standards, empty catch blocks must be commented to explain why it's acceptable to ignore exceptions.